src/ol_infrastructure/applications/ocw_build/__main__.py

```diff
@@ -0,0 +1,220 @@
+                'Action': 'sts:AssumeRole',
+                'Principal': {'Service': 'ec2.amazonaws.com'}
+            }
+        }
+    ),
+    name=f'ocw-build-instance-role-{env_suffix}',
+    path='/ol-applications/ocw-build-role'
+)
+
+iam.RolePolicyAttachment(
+    f'ocw-build-role-policy-{env_suffix}',
+    policy_arn=iam_instance_policy.arn,
+    role=iam_role.name
+)
+
+ocw_build_instance_profile = iam.InstanceProfile(
+    f'ocw-build-instance-profile-{env_suffix}',
+    role=iam_role.name,
+    name=f'ocw-build-instance-profile-{env_suffix}',
+    path='/ol-applications/ocw-build-instance-profile/'
+)
+
+# Salt minion
+
+ocw_build_minion_id = f'ocw-build-{ocw_next_build_environment}-0'
+salt_minion = OLSaltStackMinion(
+    f'saltstack-minion-{ocw_build_minion_id}',
+    OLSaltStackMinionInputs(
+        minion_id=ocw_build_minion_id,
+        salt_api_url=get_config('saltstack:api_url'),
+        salt_user=get_config('saltstack:api_user'),
+        salt_password=get_config('saltstack:api_password')
+    )
+)
+
+cloud_init_userdata = build_userdata(
+    instance_name=ocw_build_minion_id,
+    minion_keys=salt_minion,
+    minion_roles=['ocw-build'],
+    minion_environment=ocw_next_build_environment,
+    salt_host=f'salt-{env_suffix}.private.odl.mit.edu'
+)
+
+iam_image = get_ami(
+    filters=[
+        {
+            'name': 'tag:Name',
+            'values': ['ocw-build'],
+        },
+        {
+            'name': 'virtualization-type',
+            'values': ['hvm'],
+        },
+    ],
+    most_recent=True,
+    owners=[str(get_caller_identity().account_id)]
+)
+instance_tags = aws_config.merged_tags({'Name': ocw_build_minion_id})
+ec2_instance = ec2.Instance(
+    f'ocw-build-instance-{ocw_next_build_environment}',
+    ami=iam_image.id,
+    user_data=cloud_init_userdata,
+    instance_type=get_config('ocw_build:instance_type'),
+    iam_instance_profile=ocw_build_instance_profile.id,
+    tags=instance_tags,
+    volume_tags=instance_tags,
+    subnet_id=apps_vpc['subnet_ids'][0],
+    key_name=get_config('saltstack:key_name'),
+    root_block_device=ec2.InstanceRootBlockDeviceArgs(
+        volume_type='gp2',
+        volume_size=100
+    ),
+    vpc_security_group_ids=[
+        apps_vpc['security_groups']['default'],
+        apps_vpc['security_groups']['web'],
+        apps_vpc['security_groups']['salt_minion'],
+    ],
+    opts=ResourceOptions(depends_on=[salt_minion])
+)
+
+# DNS
+
+fifteen_minutes = 60 * 15
+route53_domain = route53.Record(
+    f'ocw-build-{env_suffix}-service-domain',
+    name=get_config('ocw_build:domain'),
+    type='A',
+    ttl=fifteen_minutes,
+    records=[ec2_instance.public_ip],
+    zone_id=mitodl_zone_id,
+    opts=ResourceOptions(depends_on=[ec2_instance])
+)
+route53_domain_v6 = route53.Record(
+    f'ocw-build-{env_suffix}-service-domain-v6',
+    name=get_config('ocw_build:domain'),
+    type='AAAA',
+    ttl=fifteen_minutes,
+    records=[ec2_instance.ipv6_addresses],
```

Comment:
The `ipv6_addresses` attribute is already a list

Reply:
Fixed in b259de5